README.md

```diff
@@ -249,6 +249,7 @@ The following **SAST** (*static application*/*analysis security*/*software testi
 | [Cppcheck](https://cppchecksolutions.com/)                                                                         | Static analysis tool for C and C++ code                                            |
 | [DUMA](https://github.com/johnsonjh/duma)                                                                          | Detect Unintended Memory Access, a memory debugger                                 |
 | [Flawfinder](https://dwheeler.com/flawfinder/)                                                                     | Scans C and C++ source code for potential security weaknesses                      |
+| [Funcheck](https://github.com/tmatis/funcheck)                                                                     | A tool for checking functions calls return protections                             |
```

Comment:
Did you mean function call?

Reply:
Hah.  Probably.  I actually copied that directly from their README so it's not technically my fault.